aws/cloudformation/bootstrap_standalone.sh.erb

```diff
@@ -20,7 +34,24 @@ S3_BUCKET=<%=s3_bucket%>
 REGION=<%=region%>
 INSTANCE_ID=$(curl -s http://169.254.169.254/latest/meta-data/instance-id)
 NODE_NAME=<%=rack_env%>-${INSTANCE_ID}
-ENVIRONMENT=<%=environment%>
+
+# Install AWS CloudWatch Logs
+# Ref: http://docs.aws.amazon.com/AmazonCloudWatch/latest/DeveloperGuide/EC2NewInstanceCWL.html
+AWSLOGS_AGENT_URL=https://s3.amazonaws.com/aws-cloudwatch/downloads/latest/awslogs-agent-setup.py
+AWSLOGS_CONF=${MYTMPDIR}/awslogs.conf
+cat <<CONF > ${AWSLOGS_CONF}
+[general]
+region = ${REGION}
+[${LOG}]
+log_group_name = ${STACK}
+log_stream_name = ${LOG}
+file = ${LOG}
+CONF
+curl -s ${AWSLOGS_AGENT_URL} | python - -n -r ${REGION} -c ${AWSLOGS_CONF} > /dev/null
```

Comment:
the `awslogs-agent-setup.py` script doesn't have any `-q`/`--quiet` option and is too noisy on successful installation, so redirecting its output to `/dev/null` quiets down the output log to a more comfortable level.